packages/circuit-ui/styles/style-mixins.docs.mdx

```diff
@@ -117,6 +117,25 @@ function focusOutline('inset'): (theme: Theme | { theme: Theme }) => SerializedS
 
 <Story id="features-style-mixins--focus-outline" />
 
+## `inputOutline`
+
+Visually communicates to the user that an element is hovered, focused, or active; including for the disabled, invalid and warning states.
```

Comment:
Please suggest any better ideas for wording here 🙏 

Basically the mixin handles styles for two types of states :

1. disabled/invalid/warning/default (take precedence in that order)
2. hovered/focused/active

...and I'm not sure how to best put this

Reply:
```suggestion
Visually communicates to the user that an element is hovered, focused, or active in the disabled, invalid, and warning states.
```